src/os/bluestore/BlueStore.cc

```diff
@@ -10305,8 +10305,6 @@ int BlueStore::_do_write(
   o->extent_map.compress_extent_map(dirty_start, dirty_end - dirty_start);
   o->extent_map.dirty_range(dirty_start, dirty_end - dirty_start);
 
-  r = 0;
-
  out:
```

Comment:
@ztczll This is not introduced by mistake. This is a reset statement, which `r` is reset to `0` when success. The same pattern has been followed in other places in the same file. 

Considering your change, The point is what if `_do_alloc_write()` returns a positive integer ? The original author still want to set the value of `r` to `0` ?

So I request @liewegas to comment on this change. Thanks.